libraries/AdaptiveExpressions/ExpressionFunctions.cs

```diff
@@ -3762,6 +3769,81 @@ private static IEnumerable<object> Flatten(IEnumerable<object> list, int dept)
                     },
                     ReturnType.Number,
                     expr => ValidateArityAndAnyType(expr, 1, 1, ReturnType.String)),
+                new ExpressionEvaluator(
+                    ExpressionType.TicksToDays,
+                    (expr, state, options) =>
+                    {
+                        object value = null;
+                        string error = null;
+                        IReadOnlyList<object> args;
+                        long ticksPerDay = 864000000000;
```

Comment:
>864000000000 [](start = 43, length = 12)

Don't you need an L on the end?  This is bigger than an int.  Should also be a constant and not local.

Reply:
changed